source/_patterns/04-pages/msa.mustache

```diff
@@ -24,21 +24,25 @@
       </div>
 
 
-    <div class="wrapper">
+    <div class="wrapper wrapper--listing wrapper--listing-selector">
 
       <div class="grid">
 
         <div class="listing-column-container">
 
-          <div class="grid__item large--eight-twelfths listing-column-main">
+          {{#viewSelector}}
+            {{> molecules-view-selector }}
```

Comment:
Is the view selector supposed to be on the homepage?

Reply:
Yes, it should be on Home and MSA